### ISSUE TITLE
HoVerNet validation and test data-loaders are using augmentations

### ISSUE THREAD

**Issue by eyalol** [Dana-Farber-AIOS/pathml#307]
**Describe the bug**
Hey!
I was training HoVerNet using your [notebook example](https://pathml.readthedocs.io/en/latest/examples/link_train_hovernet.html), When I noticed all data-loaders are being initialized the same, (with transforms). However, I think it's correct to use validation and test data-loaders without augmentations, A batch with hard augmentations could easily harm the validation results and prevent us from choosing a good model, that only performs a bit worse on say, very blurred images. It would be a really easy fix to do that I already did locally, which is just adding a flag of `transforms` inside each data-loader and toggling it according to the data loader type, `false` for validation and test, `true` for training. First I wanted to raise it to discussion here before opening a pull request for it. Let me know what do you think.
Thanks!

**Comment by jacob-rosenthal** [comment]
Thank you for catching this, @eyalol! I think you are right. For the validation and test sets, we want to estimate performance on "unseen" data, and any unseen data from the real world wouldn't be augmented. 

Would probably be best to add a `transforms` argument to `PanNukeDataModule._get_dataset()`:
 https://github.com/Dana-Farber-AIOS/pathml/blob/7e64b94b65f09f89e9177e7d36746c539d4812d0/pathml/datasets/pannuke.py#L236-L243

And then pass True when calling `_get_dataset()` inside `train_dataloader()` and False for `valid_dataloader()` and `test_dataloader()`:
https://github.com/Dana-Farber-AIOS/pathml/blob/7e64b94b65f09f89e9177e7d36746c539d4812d0/pathml/datasets/pannuke.py#L358-L403

If you want to open a PR that would be great. Thanks for contributing!

**Comment by eyalol** [comment]
Thanks for the response, the fix you proposed is exactly what I implemented locally. I'll get to it.
A few questions that are not so related:

* When I'm submitting a PR, it would be merged to your `dev` branch, and if I understood correctly, is different from your production branch which is used for publishing new package version. so my question is - such change won't be merged into your pip package anytime soon? I'm asking since I'm currently using pip for it, since it makes life easier with the docker.

* How do you maintain the pip package?

* Is there an estimation for pretrained model? especialy for hovernet, since I couldn't reproduce your results that would be awesome. should be pretty easy to add it. I can submit a feature request for it if needed as well.
Thanks!

**Comment by jacob-rosenthal** [comment]
Great, thank you!

Yes, our workflow is that new PRs should be merged to dev, then we can cut releases from dev to master.  We can make a new release without delay once the PR is merged.

Whenever a new release is created, we have a GitHub Action workflow which automatically builds the package and uploads it to PyPI. https://github.com/Dana-Farber-AIOS/pathml/blob/master/.github/workflows/publish-to-pypi.yml

We haven't set up a good way to publish the model weights yet. It's on the to-do list but just haven't had bandwidth. If you want to help put something together, that would be awesome!! In the meantime, here's a link to the model weights from the notebook: https://www.dropbox.com/s/rq3cqzopnlfuj4c/hovernet_pannuke.pt?dl=0

Thanks again, appreciate your help!

**Comment by Dana-Farber** [comment]
Shall we consider a pathml-nightly built that has the dev branch in??

**Comment by jacob-rosenthal** [comment]
Fixed in #309 . Thanks!
